groups/restrictions.yaml

```diff
@@ -0,0 +1,14 @@
+restrictions:
+  - path: "committee-code-of-conduct/groups.yaml"
+    allowedGroups:
+      - "^code-of-conduct@knative.team$"
+#   - path: "committee-security-response/groups.yaml"
```

Comment:
What's the reason for adding these lines and commenting them out?

Reply:
this is fixed now.